src/common/commonutils/SshUtils.c

```diff
@@ -626,8 +627,18 @@ static int CheckSshWarningBanner(char** reason, OsConfigLogHandle log)
 
     if (NULL != (bannerPath = GetSshServerState(banner, log)))
     {
-        OsConfigLogInfo(log, "CheckSshWarningBanner: '%s' found in SSH Server response set to '%s'", banner, bannerPath);
-        status = CheckFileExists(bannerPath, reason, log);
+        if (0 == strcmp(bannerPath, none))
+        {
+            OsConfigLogInfo(log, "CheckSshWarningBanner: '%s' found in SSH Server response set to '%s', which means no banner is set", banner, bannerPath);
+            OsConfigCaptureReason(reason, "'%s' is set to '%' in SSH Server response, which means no batter is set", banner, bannerPath);
```

Comment:
```suggestion
            OsConfigCaptureReason(reason, "'%s' is set to '%' in SSH Server response, which means no batter is set", none, bannerPath);
```
I am not 100% sure what was the intention, but I think it should be like this, same for the line above.